java/mykola-melnyk/src/com/zoolatech/lecture4/tasks/_1/Container.java

```diff
@@ -0,0 +1,33 @@
+package com.zoolatech.lecture4.tasks._1;
+
+public class Container {
+    private int value;
+
+    public Container (int value) {
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public void setValue(int value) {
+        this.value = value;
+    }
+
+    class ContainerUpdatesListener {
+        private int firstValue;
+
+        public ContainerUpdatesListener() {
+            Container.this.value = firstValue;
+        }
+        public boolean hasValueChanged() {
+            if (Container.this.value == firstValue) {
```

Comment:
`==` already returns a boolean value, you can simplify your code by directly returning it, without the if condition

Reply:
Done